### DISCUSSION TITLE
Are layer groups supported?

### DISCUSSION THREAD

**Discussion by baucez** [haplokuon/netDxf#464] [General]
I'm asking if layer groups (AcLyLayerGroup) are supported by the library.
I can't find any reference to it so I supposed the answer is no.

Will they be supported in the future?

**Comment by haplokuon** [comment]
No, layer filters neither, only layer states are available. Too many things are undocumented in the DXF format. I am most interested in the geometric data and not this kind of things that are very specific to the AutoCad UI.
